### PR TITLE
Tables to sections

### PR DESCRIPTION
**Pull request by dericed**
This partly resolves https://github.com/Matroska-Org/ebml-specification/issues/89. Once accepted I can make another PR to incorporate cross-sectional references and forward referencing as needed.

### CODE REVIEW

**Comment by robUx4** [comment]
LGTM.

The `maxOccurs: 1` may be omitted. But since they are only for the EBML header it's better to be safe and make it explicit.

File: specification.markdown [hunk 1]
```diff
@@ -448,116 +448,191 @@ This table clarifies if a Mandatory EBML Element MUST be written, according to i
...
+
+maxOccurs: 1
+
+range:  >3
+
+default: 4
+
+type: Unsigned Integer
+
+description: The `EBMLMaxIDLength Element` stores the maximum length in octets of the `Element IDs` to be found within the `EBML Bod`y. An `EBMLMaxIDLength Element` value of four is RECOMMENDED, though larger values are allowed.
```
Comment (retokromer) [inline specification.markdown]: typo: \`EBML Body\`

File: specification.markdown [hunk 2]
```diff
@@ -448,116 +448,191 @@ This table clarifies if a Mandatory EBML Element MUST be written, according to i
...
+
+maxOccurs: 1
+
+range:  -
+
+default: 1
+
+type: Unsigned Integer
+
+description: The version of DocType interpreter used to create the EBML Document.
```
Comment (retokromer) [inline specification.markdown]: typo? \`DocType\` and \`EBML Document\`

File: specification.markdown [hunk 3]
```diff
@@ -448,116 +448,191 @@ This table clarifies if a Mandatory EBML Element MUST be written, according to i
...
+
+minOccurs: 0
+
+maxOccurs: 1
+
+size: 4
+
+type: Binary
+
+description: The `CRC-32 Element` contains a 32 bit Cyclic Redundancy Check value of all the `Element Data` of the `Parent Element` as stored except for the `CRC-32 Element` itself. When the `CRC-32 Element` is present, the `CRC-32 Element` MUST be the first ordered `EBML Element` within its `Parent Element` for easier reading. All `Top-Level Elements` of an `EBML Document` SHOULD include a `CRC-32 Element` as a `Child Element`. The CRC in use is the IEEE-CRC-32 algorithm as used in the [@!ISO.3309.1979] standard and in section 8.1.1.6.2 of [@!ITU.V42.1994], with initial value of `0xFFFFFFFF`. The CRC value MUST be computed on a little endian bitstream and MUST use little endian storage.
```
Comment (retokromer) [inline specification.markdown]: I guess should read: contains a 32-bit Cyclic Redundancy (with -)

**Comment by robUx4** [comment]
I noticed some typos too but they were already there and it's better to go with the structure change and fix the content in subsequent PRs.

File: specification.markdown (hunk 1 above)
Comment (dericed) [inline specification.markdown]: updated and squashed into https://github.com/Matroska-Org/ebml-specification/pull/101/commits/229e83b3d18eb5dc6162343fa2f65849ee7ae4d9

File: specification.markdown (hunk 2 above)
Comment (dericed) [inline specification.markdown]: updated and squashed into https://github.com/Matroska-Org/ebml-specification/pull/101/commits/229e83b3d18eb5dc6162343fa2f65849ee7ae4d9

File: specification.markdown (hunk 3 above)
Comment (dericed) [inline specification.markdown]: updated and squashed into https://github.com/Matroska-Org/ebml-specification/pull/101/commits/229e83b3d18eb5dc6162343fa2f65849ee7ae4d9

**Comment by dericed** [comment]
updated per @retokromer's comments, ready for re-review

**Comment by retokromer** [comment]
LGTM

**Comment by mbunkus** [comment]
Thanks :)

**Pull request merged**